modules/gltf/gltf_document.cpp

```diff
@@ -5285,6 +5315,7 @@ void GLTFDocument::_convert_scene_node(Ref<GLTFState> p_state, Node *p_current,
 		gltf_root = current_node_i;
 		p_state->root_nodes.push_back(gltf_root);
 	}
+	gltf_node->merge_meta_from(p_current);
```

Comment:
I think this like should come earlier, at least above `ext->convert_scene_node`, so that extensions can have access to the metadata, and/or tweak the glTF node's metadata without affecting the Godot scene node.

In fact, I think we can move this way up, right below `gltf_node->set_name`.

Reply:
I was working on this part of the code anyway, so I put this change in PR #96468.

Reply:
Thanks